src/I18n/I18n.php

```diff
@@ -260,6 +265,40 @@ public static function defaultFormatter($name = null)
     }
 
     /**
+     * Sets the name of the default output timezone to use for future
+     * date and time operations.
+     *
+     * If called with no arguments, it will return the currently configured value if it is
+     * the same for Time, Date, FrozenTime and FrozenDate. If not it will return false.
+     *
+     * @param string|null $timezone The name of the output timezone to use.
+     * @return \DateTimezone|null|false The default output datetime zone.
+     */
+    public static function defaultOutputTimezone($timezone = null)
+    {
+        if (is_string($timezone)) {
+            $timezone = new \DateTimeZone($timezone);
+        }
+        if ($timezone !== null) {
+            Time::setDefaultOutputTimezone($timezone);
+            Date::setDefaultOutputTimezone($timezone);
+            FrozenTime::setDefaultOutputTimezone($timezone);
+            FrozenDate::setDefaultOutputTimezone($timezone);
+        } else if ($timezone === null) {
+            $t = Time::getDefaultOutputTimezone();
+            $d = Date::getDefaultOutputTimezone();
+            $ft = FrozenTime::getDefaultOutputTimezone();
+            $fd = FrozenDate::getDefaultOutputTimezone();
+            if ($t == $d && $d == $ft && $ft == $fd) {
+                return $t;
+            } else {
+                return false;
```

Comment:
If the defaultOutputTimezones were not matching for the 4 static classes. However I am not going to do that in the followup PR.